easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-CrayGNU-17.08-Python-3.5.2.eb

```diff
@@ -0,0 +1,74 @@
+# @author: Guilherme Peretti-Pezzi (CSCS)
+
+easyblock = "CMakeMake"
+
+name = 'OpenCV'
+version = '3.1.0'
+versionsuffix = '-python-3'
```

Comment:
name of the file must match the actual version, here's the error message:
```
eb CNTK-2.0.rc1-CrayGNU-17.08.eb -f -r  --modules-
ERROR: Irresolvable dependencies encountered: OpenCV/3.1.0-CrayGNU-17.08-python-3
```
I suggest you to use `versionsuffix = '-python-3'` in order to be coherent with the other .eb files using `cray-python`.

Therefore this file should be renamed to `OpenCV-3.1.0-CrayGNU-17.08-python-3.eb` and you should update the CNTK recipe as well.